### PR TITLE
Upgrade to arcgis earth v1.12

### PR DESCRIPTION
**Pull request by xuew8412**
Add four functions of Graphic:
1. AddGraphic
2. GetGraphic
3. UpdateGraphic:
4. RemoveGraphic:

Will merge it after Earth 1.12 released.
- [x] Code Review @yong7743 
- [x] Code Review @gis2all

### CODE REVIEW

File: src/ArcGISEarth.AutoAPI.Examples/MainWindowViewModel.cs [hunk 1]
```diff
@@ -211,6 +223,10 @@ public string PrettyJson(string jsonString)
         private const string GETLAYER_EXAMPLE = "311b7317-94f8-4f80-89f2-0e3ca5e77d28";
         private const string REMOVELAYER_EXAMPLE = "311b7317-94f8-4f80-89f2-0e3ca5e77d28";
         private const string REMOVELAYERS_EXAMPLE = "operationalLayers";
+        private const string ADDGRAPHIC_EXAMPLE = "{ \"geometry\": {\"type\": \"point\", \"x\": -100, \"y\": 40 }, \"symbol\": { \"type\": \"picture-marker\", \"url\": \"https://static.arcgis.com/images/Symbols/Shapes/BlackStarLargeB.png\", \"width\": \"64px\", \"height\": \"64px\", \"xoffset\": \"10px\", \"yoffset\": \"10px\"}}";
```
Comment (yong7743) [inline src/ArcGISEarth.AutoAPI.Examples/MainWindowViewModel.cs:226]: Do you need to set a graphic id to the json to ensure that you can call get graphic successfully?

Comment (xuew8412) [reply src/ArcGISEarth.AutoAPI.Examples/MainWindowViewModel.cs:226]: I think it's not necessary as there is an error response if call failed. The example is also the same as the JS example.

Comment (yong7743) [reply src/ArcGISEarth.AutoAPI.Examples/MainWindowViewModel.cs:226]: Sounds good.

File: src/ArcGISEarth.AutoAPI.Shared/AutomationAPIHelper.cs [hunk 2]
```diff
@@ -415,4 +533,28 @@ private static string GetBaseUrl()
...
+            HttpResponseMessage response = new HttpResponseMessage();
+            try
+            {
+                response = await client.SendAsync(request);
+            }
+            catch (TaskCanceledException e)
+            {
+            }
+
+            return response;
```
Comment (gis2all) [inline src/ArcGISEarth.AutoAPI.Shared/AutomationAPIHelper.cs:557]: Don't use `try...catch` here. The method `UpdateGraphic
`  that call this method have handled the error if any error occurs, this method should throw error normally.

**Review by yong7743 — APPROVED** [review]
LGTM

File: src/ArcGISEarth.AutoAPI.Shared/AutomationAPIHelper.cs (hunk 2 above)
Comment (xuew8412) [reply src/ArcGISEarth.AutoAPI.Shared/AutomationAPIHelper.cs:557]: Yes, I should throw this exception here even I used `try...catch`.

Comment (xuew8412) [reply src/ArcGISEarth.AutoAPI.Shared/AutomationAPIHelper.cs]: I removed the `try...catch` block

**Review by gis2all — APPROVED** [review]
LGTM

**Pull request merged**